### PR TITLE
remove Last Test question from AOE

### LINKED ISSUE #2349: Delete the "Last Test" question from the AOE form
### Description
In the current AOE form, we ask users if this is their first test, and if it is not we ask them when their most recent test was and what kind of test it was. This was annoying and confusing for users so we pre-populate their last test in SimpleReport.

However, searching the database for every user's last test has caused N+1 and performance issues in the past, and continues to be a non-trivial load. In following up with the powers that be at HHS we learned that this data has not been widely used at the federal level (where test results are de-identified), and it is okay to remove this last test question. For state health departments, they have identified test data that they can use to lookup a person's last test or full test history.

### Proposed solution 
Remove the last test question, and send the data to ReportStream as unknown/null.!

- [ ] AOE Question

- [ ] AOE Complete Tag (ask on entry tag)
- [ ] Remove from Ask on Entry survey in backend

### Additional context
[Screen Shot 2021-08-19 at 7.28.28 AM.png](https://images.zenhubusercontent.com/5ff8bf4767136734d905d155/9c81d604-cdfc-4516-8263-551397a13a3d)

### PR DESCRIPTION
**Pull request by zdeveloper**
## Related Issue or Background Info
- resolves #2349 

## Changes Proposed
- remove Last Test question from AOE form

## Additional Information
- Sorry for the amount of changes!

### UI
- [ ] Any changes to the UI/UX are approved by design 
- [ ] Any new or updated content (e.g. error messages) are approved by design 

### Testing
- [ ] Includes a summary of what a code reviewer should verify

### Changes are Backwards Compatible
- [ ] Database changes are submitted as a separate PR
  - [ ] Any new tables that do not contain PII are accompanied by a GRANT SELECT to the no-PHI user
  - [ ] Any changes to tables that have custom no-PHI views are accompanied by changes to those views
        (including re-granting permission to the no-PHI user if need be)
  - [ ] Liquibase rollback has been tested locally using `./gradlew liquibaseRollbackSQL` or `liquibaseRollback`
- [ ] GraphQL schema changes are backward compatible with older version of the front-end

### Security
- [ ] Changes with security implications have been approved by a security engineer (changes to  authentication, encryption, handling of PII, etc.)
- [ ] Any dependencies introduced have been vetted and discussed

## Cloud
- [ ] DevOps team has been notified if PR requires ops support
- [ ] If there are changes that cannot be tested locally, this has been deployed to our Azure `test`, `dev`, or `pentest` environment for verification

### CODE REVIEW

File: backend/src/main/java/gov/cdc/usds/simplereport/api/model/TestEventExport.java [hunk 1]
```diff
@@ -296,11 +296,6 @@ public String getOrderingProviderID() {
     return provider.map(Provider::getProviderId).orElse(null);
   }
 
-  @JsonProperty("First_test")
```
Comment (timothybest-usds) [inline backend/src/main/java/gov/cdc/usds/simplereport/api/model/TestEventExport.java:299]: Have we confirmed this change with the RS team?

**Review by timothybest-usds — CHANGES_REQUESTED** [review]
These changes are not backwards compatible and will cause issues when deploying. 

When the backend app servers are updated there will still be client's running the old version that includes the last test fields. This PR should be broken up into multiple parts.
1. Stop sending 'First_test' to RS, ignore the values passed from the FE to the backend, create a new mutation with out the last test fields.
2. Remove the first test fields from the FE and convert the test edit/submits to use the new mutation operation
3. Then after all the FE clients have been update ship an update to the graphql schema to remove the fields. This can be verified by counting the calls to editQueueItem/addPatientToQueue in Azurew and waiting until they are 0

File: backend/src/main/java/gov/cdc/usds/simplereport/api/model/TestEventExport.java (hunk 1 above)
Comment (zdeveloper) [reply backend/src/main/java/gov/cdc/usds/simplereport/api/model/TestEventExport.java:299]: who should I reach to verify?

Comment (nullflux) [reply backend/src/main/java/gov/cdc/usds/simplereport/api/model/TestEventExport.java:299]: I pinged the rs-eng channel about this – I think for now we should send null or UNK, as this field isn't required per HHS, but it always feels a little risky to change the columns we send

**Pull request closed without merging**